duckduckgo/iOS
# Subscription ITP Fixes

Task/Issue URL: https://app.asana.com/0/1204099484721401/1206585463538617/f

**Description**:
- Adds `allowedDomains` setting to Webviews to prevent the user from loading pages outside of the scope
- Opens External Iris links in a separate Sheet
- Adds privacy protection and content blocking rules to Webviews
- Minor UI glitches and fixes
- Enabled links from Subscription Welcome page to ITR and PIR



**Steps to test this PR**:
1. Get a subscription
2. Confirm the ITR link in Welcome page works
3. Go to Settings > Identity Theft Recovery 
4. Scroll down to the VERY bottom
5. Tap the link to `https://www.irisidentityprotection.com/terms-conditions` link
6. Observe a new sheet pops up
7. You should be able to navigate within the Iris website, but not to external URLs (Such as the Facebook link at the bottom)
